### ISSUE TITLE
Comments on the Readme

### ISSUE THREAD

**Issue by Mikemosca** [IBM/service-validator#5]
Hi,  Working on using the tool for use with the Watson services.  Have a few minor comments that may help the Readme.
Nice work.

<img width="1519" alt="Screen Shot 2020-09-01 at 12 44 45 PM" src="https://user-images.githubusercontent.com/20446629/92021563-f5b4dd80-ed27-11ea-80f5-4e3bb258cf8f.png">

<img width="1582" alt="Screen Shot 2020-09-01 at 1 28 19 PM" src="https://user-images.githubusercontent.com/20446629/92021581-ff3e4580-ed27-11ea-9453-6f9f75bc58c8.png">

<img width="1276" alt="Screen Shot 2020-09-01 at 1 48 35 PM" src="https://user-images.githubusercontent.com/20446629/92021603-09604400-ed28-11ea-8972-053989b2cd7e.png">

<img width="1680" alt="Screen Shot 2020-09-01 at 2 07 27 PM" src="https://user-images.githubusercontent.com/20446629/92021634-10875200-ed28-11ea-8e6f-ab854cb1cec4.png">

<img width="1465" alt="Screen Shot 2020-09-01 at 2 22 17 PM" src="https://user-images.githubusercontent.com/20446629/92021664-1a10ba00-ed28-11ea-8246-75d12e469766.png">

**Comment by rmkeezer** [comment]
Thanks @Mikemosca for the feedback, I really appreciate it. I made a few changes to clarify most of your questions and updated a few things that were outdated.

I have opened a PR for the readme changes related to this here: https://github.com/IBM/service-validator/pull/10 if you want to comment on anything before I merge it in the next few days.

Everything is pretty self explanatory in the PR but for the comments on the run command example, the reason we specify the word "run" is because we have chosen act as a wrapper around the `schemathesis` library and in order to keep all the functionality of it we modeled most of our modules around it. For this reason, we are using the same library for cli arguments, the python `click` library, and are defining the subcommands `run`, `replay`, and `init`. I could make `run` the default command but that runs into a few problems when passing in the API doc path without specifying `run`. For now, no specified command is defaulted to the `--help` command and is pretty clear on it's usage, so I won't change the way it is currently set up.
